src/main/java/org/spongepowered/api/Game.java

```diff
@@ -204,4 +205,12 @@
      */
     ChannelRegistrar getChannelRegistrar();
 
+    /**
+     * Gets the {@link ConversationManager} for managing and
+     * handling {@link org.spongepowered.api.conversation.Conversation}s.
```

Comment:
Is this standard throughout the API? It's the only place used in this class so I was not sure.